### ISSUE TITLE
[BUG] Unable to stop on any exception

### ISSUE THREAD

**Issue by jeanpierrep1972** [microsoft/vscode-dotnettools#965]
### Describe the Issue

With MAUI I am unable to debug and ask VS Code to stop where there is an exception.

I use VS Code on Mac

![image](https://github.com/microsoft/vscode-dotnettools/assets/1383636/888d6d45-eb94-4e32-82c5-94f578cda0bd)

### Environment Information

Version: 1.86.2
Commit: 903b1e9d8990623e3d7da1df3d33db3e42d80eda
Date: 2024-02-13T19:42:13.651Z
Electron: 27.2.3
ElectronBuildId: 26908389
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Darwin x64 23.2.0
| VS Bug <a href="https://devdiv.visualstudio.com/DevDiv/_queries/edit/1985678">1985678</a>

**Comment by jeanpierrep1972** [comment]
I don't have any option the break on exception in breakpoints section. Thanks

**Comment by mauroa** [comment]
Hi @jeanpierrep1972, this issue should be fixed now if you install .NET SDK 8.0.202 or higher. We've introduced a new debugger engine for MAUI in VS Code, so If you update the extension to latest pre-release and also ensure you have a compatible .NET SDK, you should be able to configure your exceptions behavior while debugging.

Let me know if that unblocks your issue. Thanks

**Comment by jeanpierrep1972** [comment]
I can confirm that it's fixed.

**Comment by dingxiaolong** [comment]
I use the dotnet version 8.0.300 on Mac , but still don't have the exception breakpoint option on my side ,  is anyone know how to solve it ?
![image](https://github.com/microsoft/vscode-dotnettools/assets/21192107/f6e10d03-10a1-46cb-aee6-7bcb873bc284)

**Comment by mauroa** [comment]
This issue has been fixed on latest .NET MAUI release version. Please ensure to also update C# and C# DevKit to latest pre-release versions as well. Thanks

**Comment by jeanpierrep1972** [comment]
I don't know why but the bug is come back..

I have the .NET SDK 8.0.302

With the .NET MAUI

		<PackageReference Include="Microsoft.Maui.Controls" Version="8.0.60" />
		<PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="8.0.60" />
		
And the extensions:

Version: 1.90.2
Commit: 5437499feb04f7a586f677b155b039bc2b3669eb
Date: 2024-06-18T22:37:41.291Z
Electron: 29.4.0
ElectronBuildId: 9728852
Chromium: 122.0.6261.156
Node.js: 20.9.0
V8: 12.2.281.27-electron.0
OS: Darwin x64 23.2.0


alefragnani.project-manager@12.8.0
aliasadidev.nugetpackagemanagergui@2.1.0
bbenoist.shell@0.3.0
cheshirekow.cmake-format@0.6.11
codezombiech.gitignore@0.9.0
dae.vscode-mac-color-picker@1.1.0
davidanson.vscode-markdownlint@0.55.0
davidmarek.jsonpath-extract@1.2.3
donjayamanne.git-extension-pack@0.1.3
donjayamanne.githistory@0.6.20
dotjoshjohnson.xml@2.5.1
eamodio.gitlens@15.1.0
editorconfig.editorconfig@0.16.4
github.remotehub@0.62.0
go2sh.cmake-integration-vscode@0.7.1
ionide.ionide-fsharp@7.20.3
jmrog.vscode-nuget-package-manager@1.1.6
jorgeserrano.vscode-csharp-snippets@1.1.0
josetr.cmake-language-support-vscode@0.0.9
k--kato.docomment@1.0.0
kaysonwu.cpptask@0.0.1
kreativ-software.csharpextensions@1.7.3
leavesster.jsonpath@0.0.7
moshfeu.compare-folders@0.24.3
ms-azure-devops.azure-pipelines@1.228.0
ms-azuretools.vscode-azureresourcegroups@0.9.1
ms-azuretools.vscode-docker@1.29.1
ms-dotnettools.csdevkit@1.7.27
ms-dotnettools.csharp@2.34.12
ms-dotnettools.dotnet-interactive-vscode@1.0.5229040
ms-dotnettools.dotnet-maui@1.1.8
ms-dotnettools.vscode-dotnet-pack@1.0.13
ms-dotnettools.vscode-dotnet-runtime@2.0.6
ms-dotnettools.vscode-dotnet-sdk@0.8.0
ms-dotnettools.vscodeintellicode-csharp@2.1.11
ms-sarifvscode.sarif-viewer@3.4.4
ms-toolsai.jupyter@2024.5.0
ms-toolsai.jupyter-keymap@1.1.2
ms-toolsai.jupyter-renderers@1.0.18
ms-toolsai.vscode-jupyter-cell-tags@0.1.9
ms-toolsai.vscode-jupyter-slideshow@0.1.6
ms-vscode-deploy-azure.azure-deploy@1.2.4
ms-vscode-remote.remote-containers@0.369.0
ms-vscode.azure-account@0.12.0
ms-vscode.azurecli@0.6.0
ms-vscode.cmake-tools@1.18.42
ms-vscode.cpptools@1.20.5
ms-vscode.hexeditor@1.10.0
ms-vscode.mono-debug@0.16.3
ms-vscode.powershell@2024.2.2
ms-vscode.remote-repositories@0.40.0
pflannery.vscode-versionlens@1.14.2
twxs.cmake@0.0.17
visualstudioexptteam.intellicode-api-usage-examples@0.2.8
visualstudioexptteam.vscodeintellicode@1.3.1
vmware.vscode-concourse@1.55.0
vscode-icons-team.vscode-icons@12.8.0
ziyasal.vscode-open-in-github@1.3.6

**Comment by jeanpierrep1972** [comment]
The bug has come back. I created this new issue:

https://github.com/microsoft/vscode-dotnettools/issues/1242
